tests/queries/0_stateless/02352_interactive_queries_from_file.expect

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/expect -f
+# tags: long, no-parallel
+
+set basedir [file dirname $argv0]
+set basename [file tail $argv0]
+
+log_user 0
+set timeout 20
+match_max 100000
+
+expect_after {
+    # Do not ignore eof from expect
+    eof { exp_continue }
+    # A default timeout action is to do nothing, change it to fail
+    timeout { exit 1 }
+}
+
+spawn bash -c "echo 'select 1;\nselect 2;\nselect 3' > queries_02352"
+spawn bash -c "source $basedir/../shell_config.sh ; \$CLICKHOUSE_CLIENT --disable_suggestion"
+expect ":) "
+
+send -- "\\i queries_02352\r"
```

Comment:
Maybe also add a case with `\i queries_02352;` as well as `\i   a file with whitespace       `

(extra whitespaces in the beginning and at the end trimmed, other preserved)